dumpling/export/status.go

```diff
@@ -48,6 +48,25 @@ func (d *Dumper) runLogProgress(tctx *tcontext.Context) {
 	}
 }
 
+type Midparams struct {
+	TotalTables       float64
+	CompletedTables   float64
+	FinishedBytes     float64
+	FinishedRows      float64
+	EstimateTotalRows float64
+}
+
+func (d *Dumper) GetParameters() (midparams *Midparams) {
+	conf := d.conf
+	mid := &Midparams{}
+	mid.TotalTables = float64(calculateTableCount(conf.Tables))
```

Comment:
To me, this parameter only needs to be computed once 🤔

Reply:
OK, I will add a judgment statement to this parameter.